chart/kubenab/templates/deployment.yaml

```diff
@@ -7,6 +7,7 @@ apiVersion: extensions/v1beta1
 kind: Deployment
 metadata:
   name: {{ include "kubenab.fullname" . }}
+  namespace: {{ .Release.Namespace | quote }}
```

Comment:
This is a good point.
Because Tools like _helmfile_ are using this flag to set the Target Namespace.
So the User (and Maintainers/Admins) don't have to maintain the _namespace_ on two places.

Reply:
Yes, you're right. I've seen namespaced resource in a lot of charts before, so I thought that it was best practice.
I take this answer from bacongobbler to be the authoritive answer regarding namespaced resources: https://github.com/helm/helm/issues/5465